blueocean-dashboard/package.json

```diff
@@ -1,71 +1,71 @@
 {
-  "name": "blueocean-dashboard",
```

Comment:
These package.json files use an indenting inconsistent with the rest of the project.  I feel like it should be fixed eventually so my editor isn't using the wrong indent every time I make a change to the files. No good? :(  Or should it just be done in a separate PR?